.goreleaser.yml

```diff
@@ -37,7 +37,7 @@ archives:
     name_template: "{{ .Binary }}-v{{ .Version }}-{{ .Os }}-{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}"
     files:
       - LICENSE
-      - README.md
+      - readme.md
```

Comment:
This should be all caps right? Should the file be renamed instead?

Reply:
I'm ok with it either way, but yeah